db/migrations/01_create _tweet_form.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE tweet_timeline(id SERIAL PRIMARY KEY, tweet VARCHAR(60))
```

Comment:
Good use of instructions on how to set up table. You should also include how to set up the database so I know it should be called 'tweet_form' and tweet_form_test'